<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QWidget" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #ADD8E6;
}
font: 9pt &quot;Microsoft YaHei UI&quot;;
font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
  </property>
  <widget class="QLineEdit" name="oldPasswordEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #FFFFFF;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="submitNewPasswordButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { font: 10px &quot;Microsoft YaHei UI&quot;, sans-serif; /* 字体大小设置为10px */ color: white; /* 文字颜色设置为白色 */ background-color: #5cb85c; /* 设置背景色为比#45a049浅的绿色 */ /* 其他样式属性 */ } QPushButton:hover { background-color: #45a049; /* 悬停时背景色变为#45a049 */ /* 悬停时若需保持其他样式不变，则无需额外设置 */ }
font: 10pt &quot;Microsoft YaHei UI&quot;;
</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="oldPasswordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>旧密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="newPasswordLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPasswordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>再次输入新密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { font: 10px &quot;Microsoft YaHei UI&quot;, sans-serif; /* 字体大小设置为10px */ color: white; /* 文字颜色设置为白色 */ background-color: #5cb85c; /* 设置背景色为比#45a049浅的绿色 */ /* 其他样式属性 */ } QPushButton:hover { background-color: #45a049; /* 悬停时背景色变为#45a049 */ /* 悬停时若需保持其他样式不变，则无需额外设置 */ }</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
